.env.example

```diff
@@ -1,4 +1,4 @@
 PORT=4173
-VITE_APP_METABASE_INSTANCE_URL=https://shoppy.hosted.staging.metabase.com
+VITE_APP_METABASE_INSTANCE_URL=https://shoppy.coredev.metabase.com
```

Comment:
When it is merged we still want to point the production app on Vercel to the old instance and switch to the new instance a bit later.

From what I see, we can do it via env vars on the Vercel